health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/CreateChecklist.js

```
@@ -201,72 +201,114 @@ const CreateChecklist = () => {
+
       codes[question.id] = code;
 
       let moduleChecklist = "hcm-checklist";
-
       let checklistTypeTemp = checklistType.toUpperCase().replace(/ /g, "_");
       let roleTemp = role.toUpperCase().replace(/ /g, "_");
       if (checklistTypeCode) checklistTypeTemp = checklistTypeCode;
-      let formattedString = `${campaignName}.${checklistTypeTemp}.${roleTemp}.${code}`;
 
+      // Format the final string with the code (generate for all questions)
+      let formattedString = `${campaignName}.${checklistTypeTemp}.${roleTemp}.${code}`;
```

Biome

[error] This let declares a variable that is only assigned once.

'formattedString' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)